packages/cfd/src/Containers/__tests__/cfd-financial-stp-real-account-signup.spec.js

```diff
@@ -0,0 +1,191 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import CFDFinancialStpRealAccountSignup from '../cfd-financial-stp-real-account-signup';
+import { localize } from '@deriv/translations';
+
+jest.mock('Stores/connect.js', () => ({
+    __esModule: true,
+    default: 'mockedDefaultExport',
+    connect: () => Component => Component,
+}));
+
+jest.mock('@deriv/account', () => ({
+    ...jest.requireActual('@deriv/account'),
+    FormSubHeader: () => <div>FormSubHeader</div>,
+}));
+
```

Comment:
since each of these components has their own test, we need just to mock them and check the header part.

Reply:
@mahdiyeh-fs there is no need to mock them if they are rendered properly, and as I can see you are not using their mocks in your tests (e.g. getByText('CFDPersoinalDetails')) but instead you're searching for real text from them directly 🙂 so the below 3 mocks are redundant